Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Header.css';
+
+export default () => {
+    return (
+        <div className="header-content">
+            @LambdaSchool - 26 jan
```

Comment:
another cool stretch would be to make that date the current date, or rather set a variable to the date that it was posted.
even better have an object called card or post with all the info and a property on it with the posted date.